### PR TITLE
Added C64 Chameleon accelerator code and documentation.

### PR DESCRIPTION
**Pull request by blackystardust**
Added C64 Chameleon accelerator code and documentation.

### CODE REVIEW

**Comment by mrdudz** [comment]
works for me :)

File: libsrc/c64/acc_detect_chameleon.s [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        sta     CHAMELEON_CFGENA
+        lda     CHAMELEON_CFGENA
+        sty     CHAMELEON_CFGENA
+        cmp     #$FF
+        beq     not_found
+found:
+        lda     #$01
+        .byte   $2C
+not_found:
+        lda     #$00
```
Comment (greg-king5) [inline libsrc/c64/acc_detect_chameleon.s]: `txa` can be used here, to return zero.

File: include/accelerator.h [hunk 2]
```diff
@@ -174,6 +174,58 @@ extern unsigned char detect_c128 (void);
  * 0x01  : C128 CPU is the current CPU
  */
 
+
+/* C64 Chameleon cartridge */
+
+extern unsigned char __fastcall__ set_chameleon_speed (unsigned char speed);
```
Comment (oliverschmidt) [inline include/accelerator.h]: Sorry for not noticing earlier :-( but we don't use "extern" for function declarations. Please remove them all here and in the comments section of the assembler source.
Additionally I ask you to remove the already merged ones - either in this pull request or a separate one, at your discretion.

**Comment by greg-king5** [comment]
:+1:

**Pull request merged**